src/System.Management.Automation/engine/parser/tokenizer.cs

```diff
@@ -1229,23 +1229,125 @@ private int SkipBlockComment(int i)
             return i;
         }
 
-        private static char Backtick(char c)
+        private string Backtick(char c)
```

Comment:
For discussion. Here we have to change both Backtick and calling code. Alternative is to return char "`u" from Backtick and then read a hex in ScanUnicodeEscapeSequence:
```c#
c = Backtick(c1);
if ( c == '\u' )
{
    ScanUnicodeEscapeSequence;
}
```